pkg/workloads/cortex/lib/storage/s3.py

```diff
@@ -249,3 +249,7 @@ def get_json_external(self, s3_path, num_retries=0, retry_delay_sec=2):
         if obj is None:
             return None
         return json.loads(obj.decode("utf-8"))
+
+    def list_objects(self, s3_path):
```

Comment:
To maintain the interface, can you implement this function in `storage/local.py` too? Also, for the implementations to match, maybe just a list of `obj["Key"]` should be returned in this case, and the file paths in the local case